inc/targetticket.class.php

```diff
@@ -1095,218 +1101,246 @@ public function save(PluginFormcreatorFormAnswer $formanswer) {
       ];
 
       $data = Ticket::getDefaultValues();
-      $ticket  = new Ticket();
       $form    = $formanswer->getForm();
       $answer  = new PluginFormcreatorAnswer();
 
-      $data['requesttypes_id'] = PluginFormcreatorCommon::getFormcreatorRequestTypeId();
-
-      // Get predefined Fields
-      $ttp                  = new TicketTemplatePredefinedField();
-      $predefined_fields    = $ttp->getPredefinedFields($this->fields['tickettemplates_id'], true);
-
-      if (isset($predefined_fields['_users_id_requester'])) {
-         $this->addActor('requester', $predefined_fields['_users_id_requester'], true);
-         unset($predefined_fields['_users_id_requester']);
-      }
-      if (isset($predefined_fields['_users_id_observer'])) {
-         $this->addActor('observer', $predefined_fields['_users_id_observer'], true);
-         unset($predefined_fields['_users_id_observer']);
+      $createTicket = false;
+      if(!empty($this->fields['has_condition'])) {
+         $conditionValues = $DB->request([
+            'SELECT' => ['answer'],
+            'FROM'   => PluginFormcreatorAnswer::getTable(),
+            'WHERE'  => [
+               'plugin_formcreator_formanswers_id' => $formanswer->fields['id'],
+               'plugin_formcreator_questions_id'   => $this->fields['condition_question']
+            ]
+         ])->next();
+         $conditionValues = $conditionValues['answer'];
+         $conditionValuesArray = preg_split("/\\r\\n/", $conditionValues);
+         foreach ($conditionValuesArray as $conditionValue) {
```

Comment:
It seems that you currently support only ORed conditions ?

```condition 1 is true OR condition 2 is true OR ... ```

In such case you can break the for loop after setting $createTicket to true